doc/conf.py

```diff
@@ -18,6 +18,8 @@
 
 html_static_path = []
 
+html_js_files = ["https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.4/require.min.js"]
```

Comment:
I am a bit surprised that we need requirejs to fix plotly plots and not just a snippet of plotly javascript. I guess it's because we are rendering them with functionality that renders to a Jupyter notebook (and Jupyter notebooks use requirejs.)